src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -420,7 +420,6 @@ func includeInSample(deviceID string, samplePercentage float64) bool {
 	}
 	// Pad and decode to int
 	paddedDeviceIDBytes := append(deviceIDBytes, 0, 0)
-	// Use LittleEndian because Mac address has most significant bytes on left
-	deviceIDInt := binary.LittleEndian.Uint64(paddedDeviceIDBytes)
+	deviceIDInt := binary.BigEndian.Uint64(paddedDeviceIDBytes)
```

Comment:
@myleshorton I found that in practice, my "fix" actually resulted in my own PC always getting sample no matter how small of a sample percentage I set.  The old "broken" algorithm didn't give us quite the right sampling percentage, but we know it's working in production because we get data for only a subset of clients, so I'm going back to that.  I added a TODO to clean this up later, but it's not entirely clear to me right now how.